immuni_exposure_reporting/helpers/validation.py

```diff
@@ -30,3 +31,20 @@ def validate_batch_index(batch_index: str) -> int:
     except ValueError:
         raise SchemaValidationException()
     return index
+
+
+def validate_batch_country(batch_country: str) -> str:
+    """
+    Validate the given batch country.
+    :param batch_country: the batch country to validate.
+    :return: the batch country, if valid.
+    :raises: SchemaValidationException if the given batch country is invalid.
+    """
+    try:
+        regex = re.compile(r"^[A-Z]{2}$", re.IGNORECASE)
```

Comment:
I'd put this outside of the method so as to reuse it every time. I'd call it something in the line of `_VALID_COUNTRY_REGEX`.